src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,42 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+export const MovieCard: React.FC<Movie> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+}) => (
+  <div className="card">
```

Comment:
remove `card` from here, card inside card - not good, you can leave here only fragment